docs/2.9.0/docs/canton/usermanual/versioning.rst

```diff
@@ -66,6 +66,8 @@ The list of supported protocol versions for each minor version is the following:
 
 .. _release-version-to-protocol-version-table:
 
+(*) The protocol versions marked with `*` is the preview protocol version. To use a preview protocol version, follow the  :ref:`documentation <how-do-i-enable-preview-features>`.
```

Comment:
Can we replace "preview protocol version" with "Initial access protocol version"?

Reply:
@hrischuk-da I agree with you that we should increase the terminology consistency. For the time being, I would prefer if we keep preview because this is aligned with the current code in Canton and Daml repository. Changing that means another round of PRs and synchronization across the two repos and currently I am not convinced that it is the best use of the resources.

In any case, we need to rediscuss the terminology for future versions in any way because "enable dev protocol versions" is not an accurate statement: dev is both a specific protocol version and a synonym of unstable. Should we schedule that for 3.0 or even 2.10 ?